packages/app-store/weather_in_your_calendar/components/AppSettings.tsx

```diff
@@ -0,0 +1,38 @@
+import { useState } from "react";
+
+import { useLocale } from "@calcom/lib/hooks/useLocale";
+import Button from "@calcom/ui/v2/core/Button";
+import { TextField } from "@calcom/ui/v2/core/form/fields";
+
+/* 
```

Comment:
Can we remove the comments? Or are they still needed for some future work?

Reply:
I think it's needed for future work. So, will keep them as is